### ISSUE TITLE
Example for multiple inputs in NXcalibration

### ISSUE THREAD

**Issue by domna** [FAIRmat-NFDI/nexus_definitions#111]

**Comment by domna** [comment]
> We can close this, right? The tutorial is part of #77.

No, there is a separate issue for discussion for each of the tutorials. #77 is just an overview issue giving a list of tutorials (also people might want to assign themselves to just writing one of these tutorials).

**Comment by lukaspie** [comment]
Got it, I overread the "Example" part and thought that this was still about the implementation of these multiple Inputs in NXcalibration 🫣
